apiserver/facades/agent/instancemutater/instancemutater_test.go

```diff
@@ -120,9 +119,9 @@ type InstanceMutaterAPILifeSuite struct {
 	instanceMutaterAPISuite
 }
 
-func TestInstanceMutaterAPILifeSuite(t *stdtesting.T) {
-	tc.Run(t, &InstanceMutaterAPILifeSuite{})
-}
+//	func TestInstanceMutaterAPILifeSuite(t *stdtesting.T) {
+//		tc.Run(t, &InstanceMutaterAPILifeSuite{})
+//	}
```

Comment:
Delete all of these comment Tests.

Reply:
Done, I went ahead and remove all tests :rofl: